### PR TITLE
Enable window-x64, s390x, ppc64le reproducible comparison

### LINKED ISSUE #743: Enable windows reproducible build comparisons to Jenkins Pipelines
Part of https://github.com/adoptium/temurin-build/issues/3245, which needs to

- Test and update reproducible compare scripts under temurin-build/tooling/
- Update tools/reproduce_comparison/Jenkinsfile
- Turn on enableReproducibleCompare

### LINKED ISSUE #758: Enable jdk17 ppc64le, s390x reproducible comparison build
Part of https://github.com/adoptium/temurin-build/issues/3245

### PR DESCRIPTION
**Pull request by sophia-guo**
jdk17 enabled.

Close #743 
Close #758 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
https://ci.adoptium.net/job/jdk17u-windows-x64-temurin_reproduce_compare/13/console

File: tools/reproduce_comparison/Jenkinsfile [hunk 1]
```diff
@@ -95,45 +99,66 @@ pipeline {
...
                     try {
                         dir('temurin-build/tooling') {
                             def rc = 0
                             if (COMPARED_JOB_NAME.contains('linux')) {
-                                rc = sh returnStatus: true, script: "./repro_compare.sh temurin ${WORKSPACE}/original temurin ${WORKSPACE}/reproduced"
+                                rc = sh returnStatus: true, script: "./repro_compare.sh temurin ${WORKSPACE}/original temurin ${WORKSPACE}/reproduced Linux"
                             } else if (COMPARED_JOB_NAME.contains('mac')) {
                                 // mac
-                                //sh "./tooling/mac_repro_compare.sh temurin ./original temurin ./reproduced ${self_cert_file} ${self_cert_passwd}"
+                                //rc = sh returnStatus: true, script: "./repro_compare.sh temurin ${WORKSPACE}/original temurin ${WORKSPACE}/reproduced Darwin"
```
Comment (karianna) [inline tools/reproduce_comparison/Jenkinsfile:156]: commented out because it's not supported yet?

**Review by karianna — CHANGES_REQUESTED** [review]

File: tools/reproduce_comparison/Jenkinsfile (hunk 1 above)
Comment (sophia-guo) [reply tools/reproduce_comparison/Jenkinsfile:156]: correct, we will work on mac later.

**Review by karianna — APPROVED** [review]

**Review by smlambert — APPROVED** [review]

**Comment by karianna** [comment]
Hmm, should code freeze bot not be blocking this?

**Review by karianna — APPROVED** [review]

**Pull request merged**